lib/Parser.js

```diff
@@ -923,6 +923,8 @@ Parser.prototype.enterObjectPattern = function enterObjectPattern(pattern, onIde
 			case "AssignmentProperty":
 				this.enterPattern(property.value, onIdent);
 				break;
+			default:
+				this.enterPattern(property.value, onIdent);
 		}
```

Comment:
Could you remove the whole switch-case and shorten this to `this.enterPattern(property.value, onIdent)`?

From AST explorer it looks like `property.type` is always `Property`.